vmware-tanzu/nsx-operator
# Add vpc attachment support

For normal vpc, operator will get the VpcConnectivityProfile path
   from nc. Only for pre created vpc, operator need to fetch the vpc
    attachment from nsx side and get VpcConnectivityProfile from vpc
    attachment

Test Done:
1. create a ns from UI, check if the attachment has been created
2. check log, if ipblockinfo controll has downloaded vpcattachment from nsx side
